src/renderer/wg_engine/tvgWgCompositor.cpp

```diff
@@ -299,7 +299,7 @@ void WgCompositor::drawShapeClipped(WgContext& context, WgRenderDataShape* rende
     endRenderPass();
     // restore current render pass
     beginRenderPass(commandEncoder, target, false);
-    RenderRegion rect = shrinkRenderRegion(renderData->aabb);
```

Comment:
@SergeyLebedkin Just want to make it sure, partial region rendering is no more after this?